controller/src/test/java/com/controller/ConfigFileParserTest.java

```diff
@@ -0,0 +1,54 @@
+package com.controller;
+
+import org.json.simple.JSONObject;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.util.HashMap;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test for ConfigFileParserTest
+ * @author Shuli
+ */
+public class ConfigFileParserTest {
+    @Before
+    public void createMockJSON() {
+        JSONObject mockObj = new JSONObject();
+        mockObj.put("database_type", "abc");
+        mockObj.put("username", "root");
+        mockObj.put("password", "12345");
+        mockObj.put("database_url", "jdbc:mysql://localhost:3306/mysqldb");
+        try {
+            PrintWriter mockJSON = new PrintWriter("./mock_config.json","UTF-8");
```

Comment:
Can you place all of the mock configuration files into their own directory?

Reply:
Done